### PR TITLE
Create v2.0.0.alpha-3.MD release notes

### PR DESCRIPTION
**Pull request by iamsamirzon**
Proposed release notes for the overall system release of Notary v2. There will be individual release notes created for the Notaryproject, Notation , Notation-go, and Notation-core-go as well.

### CODE REVIEW

File: RELEASENOTES/v2.0.0.alpha-3.MD [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+### Notation CLI Client
+Notation is a standalone CLI client based on Notary v2 specs. It supports signing, verifying, and storing signatures in [oras-artifacts-spec](https://github.com/oras-project/artifacts-spec/) compliant registries. Starting with this release of Notation, the trust store will be based on a directory structure and trust policy as a JSON document. Users can simply insert  the public certificates inside pre-configured directories for configuring their trust store, and modify the trust policy document to granularly control the scope and behavior of the signature verification policies. The use of the existing CLI commands (introduced in the alpha-1 version of Notation client when the keys/certificates for signing and verifying were specified in a file) has been subsituted by use of a directory based trust store.  Refer https://github.com/notaryproject/notation/blob/main/specs/notation-config.md.
+
+In this release of Notation command, the following CLI commands are supported. Refer (TBD - Getting started here)
+- notation sign
```
Comment (dtzar) [inline RELEASENOTES/v2.0.0.alpha-3.MD:39]: We should add support for `notation cert generate-test` https://github.com/notaryproject/notation/issues/252

**Review by dtzar — APPROVED** [review]
A couple comments, but overall looks good.

File: RELEASENOTES/v2.0.0.alpha-3.MD [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+For detailed notes on each of the individual repositories (NotaryProject, Notation (for reference CLI implementation) and Notation libraries) users can refer to the individual release notes. A summary view is provided in this document.
+
+### Notary v2 specifications
+Updated Trust Policy, Trust Store,  Signature Verification and Signing workflows along with extensibility mechanisms via Notation plugins.
+- A final JWS based [signature envelope format.](https://github.com/notaryproject/notaryproject/blob/main/signature-envelope-jws.md)
+- Updated [Signature specfication](https://github.com/notaryproject/notaryproject/blob/main/signature-envelope-jws.md) with extensibility provisions for additional signature envelope format.
+- Updated [signing and verification workflows](https://github.com/notaryproject/notaryproject/blob/main/signing-and-verification-workflow.md)
+- Introducing a new [Signing scheme](https://github.com/notaryproject/notaryproject/blob/main/signing-scheme.md) to integrate Notation Client with Signing Services
+- Updated  [trust store and trust policy](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md) specification
```
Comment (dtzar) [inline RELEASENOTES/v2.0.0.alpha-3.MD:33]: IMO - we should make sure we have documentation supporting how to use these manually.

File: RELEASENOTES/v2.0.0.alpha-3.MD [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Notary v2 Alpha 3 Release
+
+## Overview
+Notary v2 working group is pleased to announce the third release, Alpha 3. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). This release includes updates to the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications and an updated implementation of Notation client for signing and verifying artifacts.
+
+- [Notary v2 specifications](https://github.com/notaryproject/notaryproject) ***[(notaryproject-v1.0.0-draft.3)](https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-draft2)***    (Updated in this release)  
+- [Notation Library](https://github.com/notaryproject/notation-go) (https://github.com/notaryproject/notation-core-go) ***[(notation-go-v1.0.0-alpha.3)](https://github.com/notaryproject/notation-go-lib/releases/tag/v0.7.0-alpha.1)***   (Updated in this release)
+- [Notation CLI](https://github.com/notaryproject/notation)   ***([notation-v1.0.0-alpha.3)](https://github.com/notaryproject/notation/releases/tag/v0.79.0-alpha.1)***   (Updated in this release)
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.alpha-3.MD:8]: It seems the link for notation CLI is not correct. Could you correct it?

File: RELEASENOTES/v2.0.0.alpha-3.MD [hunk 4]
```diff
@@ -0,0 +1,59 @@
+# Notary v2 Alpha 3 Release
+
+## Overview
+Notary v2 working group is pleased to announce the third release, Alpha 3. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). This release includes updates to the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications and an updated implementation of Notation client for signing and verifying artifacts.
+
+- [Notary v2 specifications](https://github.com/notaryproject/notaryproject) ***[(notaryproject-v1.0.0-draft.3)](https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-draft2)***    (Updated in this release)  
+- [Notation Library](https://github.com/notaryproject/notation-go) (https://github.com/notaryproject/notation-core-go) ***[(notation-go-v1.0.0-alpha.3)](https://github.com/notaryproject/notation-go-lib/releases/tag/v0.7.0-alpha.1)***   (Updated in this release)
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.alpha-3.MD:7]: There are two libraries notation-go and notation-core-go. But here it is mixed go and core-go. I would suggest using separated links.

File: RELEASENOTES/v2.0.0.alpha-3.MD (hunk 3 above)
Comment (iamsamirzon) [reply RELEASENOTES/v2.0.0.alpha-3.MD:9]: @yizha1  - I will update the links to the release tags once they are crated in individual repos

File: RELEASENOTES/v2.0.0.alpha-3.MD (hunk 1 above)
Comment (iamsamirzon) [reply RELEASENOTES/v2.0.0.alpha-3.MD:40]: added

File: RELEASENOTES/v2.0.0.alpha-3.MD (hunk 2 above)
Comment (iamsamirzon) [reply RELEASENOTES/v2.0.0.alpha-3.MD:34]: @dtzar - Agreed. We can add a link to as part of a separate PR.

File: RELEASENOTES/v2.0.0.alpha-3.MD (hunk 4 above)
Comment (iamsamirzon) [reply RELEASENOTES/v2.0.0.alpha-3.MD]: Added

File: RELEASENOTES/v2.0.0.alpha-3.MD [hunk 5]
```diff
@@ -0,0 +1,61 @@
+# Notary v2 Alpha 3 Release
+
+## Overview
+Notary v2 working group is pleased to announce the third release, Alpha 3. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). This release includes updates to the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications and an updated implementation of Notation client for signing and verifying artifacts.
+
+- [Notary v2 specifications](https://github.com/notaryproject/notaryproject) ***[(notaryproject-v1.0.0-draft.3)](https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-draft2)***    (Updated in this release)  
+- [Notation Library 1](https://github.com/notaryproject/notation-go) ***[(notation-go-v1.0.0-alpha.3)](https://github.com/notaryproject/notation-go-lib/releases/tag/v0.7.0-alpha.1)***   (Updated in this release)
+- [Notation Library 2](https://github.com/notaryproject/notation-core-go) ***[(notation-go-v1.0.0-alpha.3)](https://github.com/notaryproject/notation-go-lib/releases/tag/v0.7.0-alpha.1)***   (Added in this release)
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.alpha-3.MD:8]: the link for notation-core-go: https://github.com/notaryproject/notation-core-go/tree/v0.1.0-alpha.3

File: RELEASENOTES/v2.0.0.alpha-3.MD [hunk 6]
```diff
@@ -0,0 +1,61 @@
+# Notary v2 Alpha 3 Release
+
+## Overview
+Notary v2 working group is pleased to announce the third release, Alpha 3. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). This release includes updates to the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications and an updated implementation of Notation client for signing and verifying artifacts.
+
+- [Notary v2 specifications](https://github.com/notaryproject/notaryproject) ***[(notaryproject-v1.0.0-draft.3)](https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-draft2)***    (Updated in this release)  
+- [Notation Library 1](https://github.com/notaryproject/notation-go) ***[(notation-go-v1.0.0-alpha.3)](https://github.com/notaryproject/notation-go-lib/releases/tag/v0.7.0-alpha.1)***   (Updated in this release)
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.alpha-3.MD:7]: The link for notation-go is https://github.com/notaryproject/notation-go/tree/v0.10.0-alpha.3

File: RELEASENOTES/v2.0.0.alpha-3.MD [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+- [Notation Library 2](https://github.com/notaryproject/notation-core-go) ***[(notation-go-v1.0.0-alpha.3)](https://github.com/notaryproject/notation-go-lib/releases/tag/v0.7.0-alpha.1)***   (Added in this release)
+- [Notation CLI](https://github.com/notaryproject/notation)   ***([notation-v1.0.0-alpha.3)](https://github.com/notaryproject/notation/releases/tag/v0.79.0-alpha.1)***   (Updated in this release)
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.alpha-3.MD:9]: The link for notation CLI is https://github.com/notaryproject/notation/releases/tag/v0.10.0-alpha.3

**Review by yizha1 — COMMENTED** [review]
Update the correct links for notation lib and CLI

File: RELEASENOTES/v2.0.0.alpha-3.MD [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+
+- [Notary v2 specifications](https://github.com/notaryproject/notaryproject) ***[(notaryproject-v1.0.0-draft.3)](https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-draft2)***    (Updated in this release)  
+- [Notation Library 1](https://github.com/notaryproject/notation-go) ***[(notation-go-v1.0.0-alpha.3)](https://github.com/notaryproject/notation-go-lib/releases/tag/v0.7.0-alpha.1)***   (Updated in this release)
+- [Notation Library 2](https://github.com/notaryproject/notation-core-go) ***[(notation-go-v1.0.0-alpha.3)](https://github.com/notaryproject/notation-go-lib/releases/tag/v0.7.0-alpha.1)***   (Added in this release)
+- [Notation CLI](https://github.com/notaryproject/notation)   ***([notation-v1.0.0-alpha.3)](https://github.com/notaryproject/notation/releases/tag/v0.79.0-alpha.1)***   (Updated in this release) 
+## Goal of the release
+- Update Notary v2 specifications 
+    - Community can use the updated specifications to develop their own implementation of Notary v2 in language/platforms of their choice, and/or contribute to Notation client development.
+    - Signature format (JWS) for RC-1.
```
Comment (FeynmanZhou) [inline RELEASENOTES/v2.0.0.alpha-3.MD:13]: I would suggest updating this goal to "support signature envelop JWS for signing and verification".

**Comment by FeynmanZhou** [comment]
@iamsamirzon @dtzar @yizha1 how about we migrate all [release notes](https://github.com/notaryproject/roadmap/tree/main/RELEASENOTES) to the blog on the Notary website? It will be more intuitive and easy to find for users.

**Review by yizha1 — COMMENTED** [review]
LGTM

**Comment by dtzar** [comment]
That's possible @iamsamirzon.  How about you fix the DCO issue, merge this, and we can figure out a plan for content on the website.

File: RELEASENOTES/v2.0.0.alpha-3.MD [hunk 9]
```diff
@@ -0,0 +1,51 @@
+# Notary v2 Alpha 3 Release
+
+## Overview
+Notary v2 working group is pleased to announce the third release, Alpha 3. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). This release includes updates to the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications and an updated implementation of Notation client for signing and verifying artifacts.
+
+- [Notation Library 1](https://github.com/notaryproject/notation-core-go) ***[(notation-go-v1.0.0-alpha.3)](https://github.com/notaryproject/notation-core-go/releases/tag/v0.1.0-alpha.3)***   (Added in this release)
```
Comment (dtzar) [inline RELEASENOTES/v2.0.0.alpha-3.MD:6]: notation-go-v1.0.0-alpha.3 --> notation-core-go-v0.1.0-alpha.3

**Review by dtzar — CHANGES_REQUESTED** [review]

**Comment by dtzar** [comment]
@iamsamirzon - I would fix the error I listed, fix the DCO check issue (it lists instructions how), and then we can merge this.

File: RELEASENOTES/v2.0.0.alpha-3.MD (hunk 9 above)
Comment (iamsamirzon) [reply RELEASENOTES/v2.0.0.alpha-3.MD]: fixed, and also added "NotaryProject" release for the specs. Also took out the mention of "directory based structure" and "JSON document based Trust policy" from this alpha-3 system release notes.

cc: @dtzar , @vaninrao10 , @priteshbandi , @yizha1

**Review by dtzar — APPROVED** [review]

**Pull request merged**